src/utils.c

```diff
@@ -20,38 +20,44 @@
 
 #include "ethUstream.h"
 #include "ethUtils.h"
+#include "uint128.h"
 #include "uint256.h"
 #include "tokens.h"
 #include "utils.h"
 
-static const unsigned char hex_digits[] =
-    {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'A', 'B', 'C', 'D', 'E', 'F'};
-
 void array_hexstr(char *strbuf, const void *bin, unsigned int len) {
     while (len--) {
-        *strbuf++ = hex_digits[((*((char *) bin)) >> 4) & 0xF];
-        *strbuf++ = hex_digits[(*((char *) bin)) & 0xF];
+        *strbuf++ = HEXDIGITS[((*((char *) bin)) >> 4) & 0xF];
+        *strbuf++ = HEXDIGITS[(*((char *) bin)) & 0xF];
```

Comment:
I may be mistaken but I think there is a builtin for this in the SDK ?

Reply:
If you mean the plugin SDK yes, but it is generated and copied from the app's code.